api/v1/model_types.go

```diff
@@ -44,6 +44,10 @@ func (m *Model) SetStatusReady(r bool) {
 	m.Status.Ready = r
 }
 
+func (m *Model) GetArtifactsStatus() ArtifactsStatus {
+	return m.Status.Artifacts
+}
```

Comment:
Is there a pref to use this function over attribute refs like: https://github.com/substratusai/substratus/pull/109/files#diff-c07787f303bc8908555dfa43fe6118a2582f65095931159c74133f1287d69732R67 ?

Reply:
Hrm, this method may be cruft left over from an interface that was removed. I will revisit.

Reply:
This method should be used when the underlying struct is not available (i.e. when it is being passed via an interface)